### DISCUSSION TITLE
i want reviews and comments!!

### DISCUSSION THREAD

**Discussion by menahem121** [OpenModelDB/open-model-database#440] [General]
Hi your site is really good but when searching for models it would be nice to sort them by most downloaded, let users add 1 to 5 stars and write comments about the models.

if there is in front of me 30 models made by random people its really hard to decide which one to use

**Comment by joeyballentine** [comment]
We've thought of some ways to potentially get around this, but adding that kind of stuff would significantly increase the cost and complexity of the site. Right now it is entirely static content. To have accounts, reviews, ratings, download stats, etc, we would need a live backend which could easily get expensive quickly. Right now all our hosting is free, and we don't make any money off the site, so increasing cost is not something we are interested in doing at the moment.

But I agree, it would be nice to have these things.

**Reply by menahem121** [reply]
> We've thought of some ways to potentially get around this, but adding that kind of stuff would significantly increase the cost and complexity of the site. Right now it is entirely static content. To have accounts, reviews, ratings, download stats, etc, we would need a live backend which could easily get expensive quickly. Right now all our hosting is free, and we don't make any money off the site, so increasing cost is not something we are interested in doing at the moment.
> 
> 
> 
> But I agree, it would be nice to have these things. 

I understand, maybe what you can do is testing the models or made them test by volunteers to make your own static selection of the best model according to use case categories
